src/util/misc/matrix.ts

```diff
@@ -112,16 +132,35 @@ export const qrDecompose = (a: TMat2D): TQrDecomposeOut => {
  * @param  {Number} [options.angle] angle in degrees
  * @return {TMat2D} transform matrix
  */
-export const calcRotateMatrix = ({ angle }: TRotateMatrixArgs): TMat2D => {
-  if (!angle) {
+export const calcRotateMatrix = ({
+  angle = 0,
+  rotation,
+}: TRotateMatrixArgs & { rotation?: TRadian }): TMat2D => {
+  if (!angle && !rotation) {
     return iMatrix;
   }
-  const theta = degreesToRadians(angle),
+  const theta = rotation ?? degreesToRadians(angle),
     cosin = cos(theta),
     sinus = sin(theta);
   return [cosin, sinus, -sinus, cosin, 0, 0];
 };
 
+export const calcShearMatrix = ({
+  skewX,
+  skewY,
+  shearX,
+  shearY,
+}: {
+  skewX?: TDegree;
+  skewY?: TDegree;
+  shearX?: number;
+  shearY?: number;
+}) =>
+  multiplyTransformMatrices(
+    [1, 0, shearX ?? (skewX ? Math.tan(degreesToRadians(skewX)) : 0), 1, 0, 0],
+    [1, shearY ?? (skewY ? Math.tan(degreesToRadians(skewY)) : 0), 0, 1, 0, 0]
+  );
```

Comment:
unused? also what is the difference between skew and shear ( in english or in geometry ) ?
I thought they were the same thing

Reply:
shear is `tan(skew)`